prow/cluster/jobs/istio-releases/daily-release/istio-releases.daily-release.release-1.1.yaml

```diff
@@ -0,0 +1,158 @@
+job_template: &job_template
+  branches:
+  - "^release-1.1$"
+  decorate: true
+
+istio_container: &istio_container
```

Comment:
nit: Like `job_template` this name doesn't seem to be descriptive enough. Many of the jobs don't use this container spec.